### PR TITLE
Fixes orbit panel shows a mob twice

### PR DESCRIPTION
**Pull request by EvilDragonfiend**
## About The Pull Request
Fixes orbit panel shows a mob twice

This is because a mob is added to `mob_list` and `poi_list` at the same time when its `var/playable` is set to TRUE in a mob type (i.e. xeno)
a mob that's added to `poi_list` will not be removed because a mob key is automatically given by code but it's not occupied by players. when it's occupied by player action, a playable mob is removed from `poi_list`, but xenos are spawned then force-occupied.
I added a new proc `delayed_set_playable()` which will be used in mob init().

## Why It's Good For The Game
bug fix

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

</details>

## Changelog
:cl:
fix: Fixed orbit panel shows a mob twice
/:cl:

### CODE REVIEW

File: code/modules/mob/living/living_sentience.dm [hunk 1]
```diff
@@ -43,13 +43,26 @@
 		to_chat(src, "<span class='notice'>[get_spawner_flavour_text()]</span>")
 	return TRUE
 
+// this exists because some 'playable=TRUE' mobs are not actually playable because mob key is automatically given
+// it prevents 'GLOB.poi_list' being glitched. without this, it will show xeno(or some mobs) twice in orbit panel.
+/mob/living/proc/delayed_set_playable(called_again=FALSE)
+	set waitfor = 0 // I didn't want to use 'addtimer(CALLBACK(), 2 SECONDS)' so I tried this instead
```
Comment (itsmeow) [inline code/modules/mob/living/living_sentience.dm]: i don't like this proc, use addtimer or move the proccall after key assignment

**Review by itsmeow — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/living_sentience.dm (hunk 1 above)
Comment (EvilDragonfiend) [reply code/modules/mob/living/living_sentience.dm]: change to addtimer.

File: code/modules/mob/living/living.dm [hunk 2]
```diff
@@ -19,7 +19,7 @@
 	GLOB.mob_living_list += src
 	initialize_footstep()
 	if (playable)
-		set_playable()	//announce to ghosts
+		delayed_set_playable()	//announce to ghosts
```
Comment (itsmeow) [inline code/modules/mob/living/living.dm]: you don't need a new proc for something done only once

**Review by itsmeow — CHANGES_REQUESTED** [review]

**Comment by EvilDragonfiend** [comment]
review addressed

**Review by itsmeow — APPROVED** [review]

**Pull request merged**